File: server/Models/Account.cs

namespace fs_template.Models
{
  public class Account
  {
    [Key]
    [Required]
    public required string Id { get; set; }
    public string? Name { get; set; }
    [Required]
    public required string Email { get; set; }

    public string? Picture { get; set; }
  }
}

File: client/src/models/Account.ts
export class Account {
  id: string;
  email?: string;
  name: string;
  picture: string;

  constructor(data: {
    id: string;
    email?: string;
    name: string;
    picture: string;
  }) {
    this.id = data.id;
    this.email = data.email;
    this.name = data.name;
    this.picture = data.picture;
  }
}
